src/zenml/client.py

```diff
@@ -7624,23 +7642,36 @@ def delete_api_key(
             api_key_name_or_id=api_key.id,
         )
 
-    #############################################
-    # Tags
-    #
-    # Note: tag<>resource are not exposed and
-    # can be accessed via relevant resources
-    #############################################
-
-    def create_tag(self, tag: TagRequest) -> TagResponse:
+    # ---------------------------------- Tags ----------------------------------
+    def create_tag(
+        self,
+        name: str,
+        exclusive: bool = False,
+        color: Optional[Union[str, ColorVariants]] = None,
+    ) -> TagResponse:
         """Creates a new tag.
 
         Args:
-            tag: the Tag to be created.
+            name: the name of the tag.
+            exclusive: the boolean to decide whether the tag is an exclusive tag.
+                An exclusive tag means that the tag can exist only for a single:
+                    - pipeline run within the scope of a pipeline
+                    - artifact version within the scope of an artifact
+                    - run template
+            color: the color of the tag
 
         Returns:
             The newly created tag.
         """
-        return self.zen_store.create_tag(tag=tag)
+        request_model = TagRequest(name=name, exclusive=exclusive)
+
+        if color is not None:
+            if isinstance(color, ColorVariants):
```

Comment:
I think this can be simplified to just `request_model.color = ColorVariants(color)`, which works also for already existing enum values.

Reply:
Simplified it.